run_codenarc.py

```diff
@@ -262,6 +263,15 @@ def parse_args(args, default_jar_versions):
         help='Path to Groovylint resources directory.',
     )
 
+    arg_parser.add_argument(
+        '--single-file',
+        help=(
+            ' When given, copy this file to a temporary directory and lint it. This may'
```

Comment:
```suggestion
            'When given, copy this file to a temporary directory and lint it. This may'
```
Sorry, I added that extraneous space in my previous suggestion by accident.